From 6886c2956f61ba6fa4627ce9f5fc9e968e3e8ed7 Mon Sep 17 00:00:00 2001
From: Robin Gong <b38343@freescale.com>
Date: Tue, 22 Sep 2015 14:36:51 +0800
Subject: [PATCH 0466/5242] MLK-11407-5: ARM: dts: imx6qdl-sabresd: add WDOG_B
 reset

commit  7381fc4a471ebd63813e6aa1202bea21c479fbe5 from
https://source.codeaurora.org/external/imx/linux-imx.git

On imx6qdl-sabresd board we use WDOG2 as WDOG_B reset source. So use WDOG2
instead of WDOG1.

Signed-off-by: Robin Gong <b38343@freescale.com>

This was already done by upstream, only leftover change in 4.14 is
iomux.

Signed-off-by: Leonard Crestez <leonard.crestez@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx6qdl-sabresd.dtsi |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/imx6qdl-sabresd.dtsi b/arch/arm/boot/dts/imx6qdl-sabresd.dtsi
index be0d4a5d..89f6911 100644
--- a/arch/arm/boot/dts/imx6qdl-sabresd.dtsi
+++ b/arch/arm/boot/dts/imx6qdl-sabresd.dtsi
@@ -871,7 +871,7 @@
 
 		pinctrl_wdog: wdoggrp {
 			fsl,pins = <
-				MX6QDL_PAD_GPIO_1__WDOG2_B		0x1b0b0
+				MX6QDL_PAD_GPIO_1__WDOG2_B 0x80000000
 			>;
 		};
 	};
-- 
1.7.9.5

